docs/monitoring/infrastructure.md

```diff
@@ -0,0 +1,49 @@
+title: Sematext Core Infrastructure Monitoring
+description: Sematext Infrastructure Monitoring gives you insight into your whole infrastructure automatically based on the applications, containers, servers and general infrastructure you are monitoring.
+
+
+Points to cover:
+
+## Overview
+
+Sematext Monitoring provides complete insight into your infrastructure - applications, servers, containers, processes, inventory, events...
+
+The centerpiece is newly introduced Infra App which acts as a container for all infrastructural data about your architecture. As with other App types, you can create as many Infra Apps under your account as you wish. Each Infra App would typically be used to gather data from one environment (for example, one Infra App for production environment, one for test environment and one for development environment).
+
+SematextCloud UI will automatically create one Infra App named "Default Infra App" the first time you might need it. You can also create Infra App on your own under Integrations->Overview.
+
+## How it works
+
+New Golang-based [Sematext agent](../agents/sematext-agent) (which is part of [spm-client](../agents/spm-client) installer) will be installed on your machine. It is sending all infrastructure data and metrics to token of one of your Infra Apps. This token is configured as part of installation instructions you can find in SematextCloud UI. It can be installed either on its own (in which case you would have only Infra monitoring enabled on particular machine) or while installing one of Sematext App Agents (for example Kafka or Elasticsearch agent) - SematextCloud UI always shows the instructions which include Infra App token.
+
+Infra App can also be used to gather data about your containerized environments. Installation instructions for SPM Docker also include Infra App token, so by installing Sematext Docker agent, you will automatically install Infra App monitoring along.
```

Comment:
Infra **Apps** * can also be used to gather data about your containerized environments. Installation instructions for **Docker Monitoring** also include **an** Infra App token **. B** y installing Sematext Agent for Docker, you will automatically install Infra App Monitoring **alongside it.**

Reply:
Infra App can also be used to gather data  => Infra App also includes data
Infra App Monitoring => there is no such thing as "Infra App Monitoring".  Maybe we want to say just "When installing Sematext Agent for Docker an Infra App will be created automatically"?

Reply:
> there is no such thing as "Infra App Monitoring"

Technically there is - people will now be able to monitor just their Infra and nothing else (that old idea of providing OS-only monitoring now becomes possible through Infra app).

Will reword it this as suggested anyway.